developer_manual/digging_deeper/dashboard.rst

```diff
@@ -147,6 +147,21 @@ however this method is deprecated and will be removed once Nextcloud 19 is EOL.
         }
     }
 
+++++++++++++++++++++++++++++++++
```

Comment:
I don't think there should be a line above the heading https://sublime-and-sphinx-guide.readthedocs.io/en/latest/topics.html?highlight=heading#use-headings

Reply:
Copied from below in the same document
https://docs.nextcloud.com//server/latest/developer_manual/digging_deeper/dashboard.html#the-ioptionwidget-interface

But yeah let me remove